backend/src/main/java/com/github/mariakohnen/packbag/backend/controller/status/ControllerAdvice.java

```diff
@@ -0,0 +1,28 @@
+package com.github.mariakohnen.packbag.backend.controller.status;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.time.LocalDateTime;
+
+@org.springframework.web.bind.annotation.ControllerAdvice
```

Comment:
Als wir das im Kurs durchgenommen haben, muss ich wohl gerade Windpocken gehabt haben ....
Sieht aber ansonsten sehr schön aus ;-)

Reply:
haha, ich glaub nicht. Das ist das Errorhandling. Wir hatten die Klasse nur anders genannt und etwas weniger Infos in der ErrorResponse.